### ISSUE TITLE
Upsample node does not support align_corners attribute

### ISSUE THREAD

**Issue by peteryang1** [onnx/onnx#1612]
Hi!
While I'm converting a pytorch model to onnx, the pytorch model has a interpolate node like below:
```Python
torch.nn.functional.interpolate(x, scale_factor=2, mode='bilinear', align_corners=True)
```
this function is the same as the upsample node in onnx, but the upsample node in onnx does not support align_corners.
do we have any plan in supporting this attribute?

**Comment by mathmanu** [comment]
When I evaluate an onnx model on pytorch, I get lower accuracy that when I evaluate the checkpoint directly in pytorch. The onnx model was exported from pytorch.

Does the onnx based inference (uses caffe2 backed of pytorch) use a different upsample function compared to what pytorch uses. From a look at the source code, it appears that the caffe2 upsample functionality uses the older behaviour of pytorch (align_corners=True).

Is there a way to use the pytorch upsample functionality when using onnx inferencing. In other words, I would like to use pytorch backend when doing onnx infernece - not caffe2 backend. Please help.

**Comment by yxchng** [comment]
Any updates on this?

**Comment by tkclimb** [comment]
I encountered the sample problem, is there still no update??

**Comment by tkclimb** [comment]
I tried to convert a network including the above layer and succeeded to properly export onnx by just passing opset version to the like  function`torch.onnx.export(..., opset_version=11, ...)` .

**Comment by mathmanu** [comment]
We are on opset_version 9 and for us it is not easy to move opset version as there are dependencies outside of just training on PC. We would prefer a solution that solves it in opset_version 9 as well. 
@tkclimb, what exactly do you mean by properly export? Do you ahve scale_factor in the exported Upsample module in ONNX graph?
